packages/integrations/node/src/preview.ts

```diff
@@ -36,11 +36,20 @@ const preview: CreatePreviewServer = async function ({ client, serverEntrypoint,
 		});
 	};
 
+	const baseWithoutTrailingSlash: string = base.endsWith('/') ? base.slice(0, base.length - 1) : base;
+	function removeBase(pathname: string): string {
+		if(pathname.startsWith(base)) {
+			return pathname.slice(0, baseWithoutTrailingSlash.length);
+		}
```

Comment:
```suggestion
		if(pathname.startsWith(base)) {
			return pathname.slice(baseWithoutTrailingSlash.length);
		}
```

Should the `0` be removed if we're removing the leading base? Strange that the tests are passing

Reply:
Oh yeah, I bet since this is preview it's probably not tested.